### DISCUSSION TITLE
Welcome to expr-eval Discussions!

### DISCUSSION THREAD

**Discussion by silentmatt** [silentmatt/expr-eval#240] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by satishnalla** [comment]
we have requirement to evaluate expressions dynamically on tabular kind of data (row & columns) where columns represent variables in the expression and rows represent data values ...Is **expr-eval** library support this kind of requirement

**Comment by msouther-lisc** [comment]
Thank you for this excellent library!

I'm working on a custom form tool that uses `expr-eval` for dynamic expressions (conditional display of form elements, validation criteria, default values, etc). Because this tool will mostly be used by non-technical business users, I put together a demo app to help them better understand the expression syntax, and I thought I'd share. Here it is on StackBlitz: https://stackblitz.com/edit/t1nko8?file=index.html
